File: src/main/java/no/brreg/regnskap/jena/SCHEMA.java
package no.brreg.regnskap.jena;

import org.apache.jena.rdf.model.Model;
import org.apache.jena.rdf.model.ModelFactory;
import org.apache.jena.rdf.model.Property;


public class SCHEMA {

    public static String getURI() {
        return "http://schema.org/";
    }

    private static final Model model = ModelFactory.createDefaultModel();

    public static final Property currency = model.createProperty(getURI(), "currency");
    public static final Property startDate = model.createProperty(getURI(), "startDate");
    public static final Property endDate = model.createProperty(getURI(), "endDate");

}

File: src/main/kotlin/no/regnskap/jena/SCHEMA.kt
package no.regnskap.jena

import org.apache.jena.rdf.model.ModelFactory

object SCHEMA {

    val uri = "http://schema.org/"

    private val model = ModelFactory.createDefaultModel()

    val currency = model.createProperty(uri, "currency")
    val startDate = model.createProperty(uri, "startDate")
    val endDate = model.createProperty(uri, "endDate")

}